packages/commonwealth/client/scripts/views/modals/edit_topic_modal.tsx

```diff
@@ -43,7 +44,8 @@ export const EditTopicModal = ({
   } = topic;
 
   const navigate = useCommonNavigate();
-
+  const { mutateAsync: editTopic } = useEditTopicMutation();
+  const { mutateAsync: deleteTopic } = useDeleteTopicMutation();
```

Comment:
we should follow this pattern, so destructure `mutate` or `mutateAsync` and right away change var name for corresponding function name